flink-streaming-connectors/flink-connector-kafka-base/src/test/java/org/apache/flink/streaming/connectors/kafka/KafkaShortRetentionTestBase.java

```diff
@@ -172,8 +173,14 @@ public void cancel() {
 
 		// ----------- add consumer dataflow ----------
 
+		// the consumer should only poll very small chunks
+		Properties consumerProps = new Properties();
+		consumerProps.putAll(standardProps);
+		consumerProps.putAll(secureProps);
+		consumerProps.setProperty("fetch.message.max.bytes", "100");
```

Comment:
I think we shouldn't be setting `fetch.message.max.bytes` here. The config key for this setting has changed across Kafka versions (for 0.9+ it's `max.partition.fetch.bytes`). The version-specific `standardProps` already set values for this config.

So, the original `props` that only contains `standardProps` and `secureProps` should be enough for the test to work.